src/Theme.cpp

```diff
@@ -0,0 +1,411 @@
+    Color("Text", ImGuiCol_Text),
+    Color("TextDisabled", ImGuiCol_TextDisabled),
+    Color("WindowBg", ImGuiCol_WindowBg),
+    Color("ChildBg", ImGuiCol_ChildBg),
+    Color("PopupBg", ImGuiCol_PopupBg),
+    Color("Border", ImGuiCol_Border),
+    Color("BorderShadow", ImGuiCol_BorderShadow),
+    Color("FrameBg", ImGuiCol_FrameBg),
+    Color("FrameBgHovered", ImGuiCol_FrameBgHovered),
+    Color("FrameBgActive", ImGuiCol_FrameBgActive),
+    Color("TitleBg", ImGuiCol_TitleBg),
+    Color("TitleBgActive", ImGuiCol_TitleBgActive),
+    Color("TitleBgCollapsed", ImGuiCol_TitleBgCollapsed),
+    Color("MenuBarBg", ImGuiCol_MenuBarBg),
+    Color("ScrollbarBg", ImGuiCol_ScrollbarBg),
+    Color("ScrollbarGrab", ImGuiCol_ScrollbarGrab),
+    Color("ScrollbarGrabHovered", ImGuiCol_ScrollbarGrabHovered),
+    Color("ScrollbarGrabActive", ImGuiCol_ScrollbarGrabActive),
+    Color("CheckMark", ImGuiCol_CheckMark),
+    Color("SliderGrab", ImGuiCol_SliderGrab),
+    Color("SliderGrabActive", ImGuiCol_SliderGrabActive),
+    Color("Button", ImGuiCol_Button),
+    Color("ButtonHovered", ImGuiCol_ButtonHovered),
+    Color("ButtonActive", ImGuiCol_ButtonActive),
+    Color("Header", ImGuiCol_Header),
+    Color("HeaderHovered", ImGuiCol_HeaderHovered),
+    Color("HeaderActive", ImGuiCol_HeaderActive),
+    Color("Separator", ImGuiCol_Separator),
+    Color("SeparatorHovered", ImGuiCol_SeparatorHovered),
+    Color("SeparatorActive", ImGuiCol_SeparatorActive),
+    Color("ResizeGrip", ImGuiCol_ResizeGrip),
+    Color("ResizeGripHovered", ImGuiCol_ResizeGripHovered),
+    Color("ResizeGripActive", ImGuiCol_ResizeGripActive),
+    Color("Tab", ImGuiCol_Tab),
+    Color("TabHovered", ImGuiCol_TabHovered),
+    Color("TabActive", ImGuiCol_TabActive),
+    Color("TabUnfocused", ImGuiCol_TabUnfocused),
+    Color("TabUnfocusedActive", ImGuiCol_TabUnfocusedActive),
+    Color("DockingPreview", ImGuiCol_DockingPreview),
+    Color("DockingEmptyBg", ImGuiCol_DockingEmptyBg),
+    Color("PlotLines", ImGuiCol_PlotLines),
+    Color("PlotLinesHovered", ImGuiCol_PlotLinesHovered),
+    Color("PlotHistogram", ImGuiCol_PlotHistogram),
+    Color("PlotHistogramHovered", ImGuiCol_PlotHistogramHovered),
+    Color("TableHeaderBg", ImGuiCol_TableHeaderBg),
+    Color("TableBorderStrong", ImGuiCol_TableBorderStrong),
+    Color("TableBorderLight", ImGuiCol_TableBorderLight),
+    Color("TableRowBg", ImGuiCol_TableRowBg),
+    Color("TableRowBgAlt", ImGuiCol_TableRowBgAlt),
+    Color("TextSelectedBg", ImGuiCol_TextSelectedBg),
+    Color("DragDropTarget", ImGuiCol_DragDropTarget),
+    Color("NavHighlight", ImGuiCol_NavHighlight),
+    Color("NavWindowingHighlight", ImGuiCol_NavWindowingHighlight),
+    Color("NavWindowingDimBg", ImGuiCol_NavWindowingDimBg),
+    Color("ModalWindowDimBg", ImGuiCol_ModalWindowDimBg),
+};
+
+// Internal Load/Dump functions
+
+static const std::string SchemaVersionLabel = "ImGuiThemeVersion";
```

Comment:
use anonymous namespace instead of static please (again, i see this a bunch, wont fill whole review with these... please change all instances in your changes)

Reply:
on top, better would probably be to use `constexpr auto SchemaVersionLabel  = "ImGuiThemeVersion"s` or maybe even switch to string_view for these (we use C++20, lets use it x) )

Reply:
little note in case you dont know - ""s -> string literal, so you can look up how it is used.
string_view has ""sv.

note that ""s also means "seconds" when chrono literals are used. dont know if we by a chance use them in some header or something, so just mentioning that there may be some conflict with these suffixes.